project.clj

```diff
@@ -1,6 +1,6 @@
 (def ks-version "1.0.0")
 
-(defproject puppetlabs/trapperkeeper "1.2.1-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper "1.3.0-SNAPSHOT"
```

Comment:
Are you thinking this will be a Y version bump b/c of the introduction of the core.async dependency?